Java/ds/DetectLoopLL.java

```diff
@@ -0,0 +1,92 @@
+import java.util.*;
+import java.io.*;
+import java.lang.*;
+//nodes of Linked list
+class Node
+{
+    int data;
+    Node next;
+    Node(int x)
+    {
+        data = x;
+        next = null;
+    }
+}
+class DetectLoopLL
```

Comment:
Add access identifier to the main class

Reply:
Updated @siddharth25pandey  kindly review it.